src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/New-AzsComputeQuota.ps1

```diff
@@ -63,6 +63,14 @@ function New-AzsComputeQuota {
         [int32]
         $VirtualMachineCount = 100,
 
+        [Parameter(Mandatory = $false)]
+        [int32]
+		$MaxAllocationStandardManagedDisksAndSnapshots = 2048,
```

Comment:
Please shorten these.

Reply:
This has been discussed and PM has opened a bug to track this and get done.